main.go

```diff
@@ -86,42 +86,54 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func attachAndBroadcastToTangle(indexReq *indexRequest) {
-	provider := os.Getenv("PROVIDER")
+	// provider := os.Getenv("PROVIDER")
 	// minDepth, _ := strconv.ParseInt(os.Getenv("MIN_DEPTH"), 10, 64)
-	minWeightMag, _ := strconv.ParseInt(os.Getenv("MIN_WEIGHT_MAGNITUDE"), 10, 64)
+	// api := giota.NewAPI(provider, nil)
+	minWeightMag, _ := strconv.ParseInt(os.Getenv("MIN_WEIGHT_MAGNITUDE"), 10, 0)
 
 	// Convert []Trytes to []Transaction
-	txs := make([]giota.Transaction, len(indexReq.Trytes))
-	for i, t := range indexReq.Trytes {
-		tx, _ := giota.NewTransaction(t)
-		txs[i] = *tx
-	}
+	// txs := make([]giota.Transaction, len(indexReq.Trytes))
+	// for i, t := range indexReq.Trytes {
+	// 	tx, _ := giota.NewTransaction(t)
+	// 	txs[i] = *tx
+	// }
 
-	api := giota.NewAPI(provider, nil)
 	// _, pow := giota.GetBestPoW()
 
-	attachReq := giota.AttachToTangleRequest{
-		Command:            "attachToTangle",
-		TrunkTransaction:   indexReq.TrunkTransaction,
-		BranchTransaction:  indexReq.BranchTransaction,
-		MinWeightMagnitude: minWeightMag,
-		Trytes:             txs,
-	}
+	// attachReq := giota.AttachToTangleRequest{
+	// 	Command:            "attachToTangle",
+	// 	TrunkTransaction:   indexReq.TrunkTransaction,
+	// 	BranchTransaction:  indexReq.BranchTransaction,
+	// 	MinWeightMagnitude: minWeightMag,
+	// 	Trytes:             txs,
+	// }
 
-	attachRes, err := api.AttachToTangle(&attachReq)
-	if err != nil {
-		raven.CaptureError(err, nil)
-		return
+	// attachRes, err := api.AttachToTangle(&attachReq)
+	// if err != nil {
+	// 	raven.CaptureError(err, nil)
+	// 	return
+	// }
+
+	// Convert []Trytes to []Transaction
+	txs := make([]giota.Transaction, len(indexReq.Trytes))
+	for i, t := range indexReq.Trytes {
+		powT, err := giota.PowSSE(t, int(minWeightMag))
```

Comment:
@rfornea [WARNING] this wouldn't compile on my laptop. Not sure if the lib was doing something fancy to not compile this if my architecture does not support SSE?